projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Load the google fonts by the new Font Library. See https://wp.me/pNEWy-hhx.
```

Comment:
Let's switch to shorthand:

```suggestion
 * Load the google fonts by the new Font Library. See pNEWy-hhx-p2.
```